src/core/trulens/experimental/otel_tracing/core/init.py

```diff
@@ -0,0 +1,54 @@
+import logging
+
+from opentelemetry import trace
+from opentelemetry.sdk.resources import Resource
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import ConsoleSpanExporter
+from opentelemetry.sdk.trace.export import SimpleSpanProcessor
+from trulens.core.session import TruSession
+from trulens.experimental.otel_tracing.core.exporter import (
+    TruLensDBSpanExporter,
+)
+
+TRULENS_SERVICE_NAME = "trulens"
+
+
+logger = logging.getLogger(__name__)
+
+
+def init(session: TruSession, debug: bool = False):
```

Comment:
wait is the idea that the user will call this function?

If so, I think it's better to just have some arg in `TruSession` when you create it. Like `initialize_otel` or something.

That said, if you think this is just for testing that's fine with me.

Reply:
for now I'm hoping to have this for testing. The next steps are:
- deprecate the old experimental otel flag + its integration everywhere in `TruSession`
- introduce the new OTEL exporters into `TruSession`